### DISCUSSION TITLE
Donate Akash Terraform Provider to Akash Network Organization

### DISCUSSION THREAD

**Discussion by cloud-j-luna** [akash-network/community#502] [Integrations]
Akash Terraform Provider was developed almost 2 years ago as a way to automate deployments on the network. It has since been downloaded +10,000 times. It has been stable for quite some time as it directly uses whichever the system's Akash binary thus is compatible with every mainnet version. Currently it is published in the registry under my name, which makes it harder to find compared to having an official Terraform Registry Account for the Akash Network Organization. This discussion aims to streamline how the donation process will look like.

I'm available to perform this in sync with someone from the Overclock team. Moving the repository to the Akash Network Organization would make it easier to find, as well as contribute. No budget is to be allocated to this effort nor expected. 

## Links
[Terraform Registry](https://registry.terraform.io/providers/cloud-j-luna/akash/0.0.8)
[Repository](https://github.com/cloud-j-luna/terraform-provider-akash)

**Comment by gosuri** [comment]
Thank you @cloud-j-luna for this amazing work!

You've done a great job with the TF provider and I propose community reward at-least 5,000 AKT upon open-sourcing for the work rendered so far.

**Reply by cloud-j-luna** [reply]
I appreciate the kind offer. 😄 But I must say that my work on the TF provider was already funded by the community pool and thus I believe I got my fair share and donating it to the Akash network Organization is a way of giving it back to the community. I'll still be developing it further (with the open sourcing of the go library soon) as we aim to refactor the provider to start using the library. For anyone interested in supporting further we always welcome delegations to our Akash Validator (Quasarch) 🙏🏻

**Reply by gosuri** [reply]
That's great to hear!

**Comment by shimpa1** [comment]
@cloud-j-luna you are the best example of what Akash is all about.
Props to you. If you need help with whatever you know where to ask.

**Comment by brewsterdrinkwater** [comment]
Thank you @cloud-j-luna . I echo what all others have said in this thread. 

We can take some time during sig-clients this month to discuss the transfer, and take next steps. Thank you for all that you do.

**Comment by rodri-r** [comment]
@cloud-j-luna I am out of words, I can only say respect!

**Comment by brewsterdrinkwater** [comment]
This is complete. Thank you to @cloud-j-luna and team. 

https://github.com/akash-network/terraform-provider-akash
